docs/docs/configuration/monitoring.md

```diff
@@ -0,0 +1,15 @@
+# Monitoring
+## Ensure all topic partitions are assigned to consumer group in subscription
+
+To use this monitoring you have to enable it in file `application.yaml` in module `hermes-management`. 
+The subscriptions that have unassigned partitions are available through endpoint `/monitoring/consumergroup`.
+
+```yaml
+monitoringConsumerGroups:
+  enabled: false
+  numberOfThreads: 6
+  secondsBetweenScans: 120
```

Comment:
maybe just use Duration in the java properties file ? Then you can declare the value here as: `scanEvery: 120s` or `scanEvery: 5m`